adm/__make_nginx_conf

```diff
@@ -83,20 +105,22 @@ def get_conf(plugin_configuration_file):
                                                    "extra_nginx_conf_filename")
             if extra_nginx_conf_filename != "null":
                 extra_nginx_conf_filepath = \
-                    os.path.join(os.path.dirname(plugin_configuration_file),
+                    os.path.join(plugin_directory,
                                  extra_nginx_conf_filename)
                 with open(extra_nginx_conf_filepath, "r") as f:
                     extra_nginx_conf_string = f.read()
+                    f.close()
         extra_nginx_conf_static_string = ""
         if parser.has_option(section, "extra_nginx_conf_static_filename"):
             extra_nginx_conf_static_filename = \
                 parser.get(section, "extra_nginx_conf_static_filename")
             if extra_nginx_conf_static_filename != "null":
                 extra_nginx_conf_static_filepath = \
-                    os.path.join(os.path.dirname(plugin_configuration_file),
+                    os.path.join(plugin_directory,
                                  extra_nginx_conf_static_filename)
                 with open(extra_nginx_conf_static_filepath, "r") as f:
                     extra_nginx_conf_static_string = f.read()
+                    f.close()
```

Comment:
this line is not necessary, please remove it (with "with" statement, the close is automatic)